tests/nn/test_inference.py

```diff
@@ -76,6 +81,51 @@ def test_pipeline(test_labels):
     return p
 
 
+@pytest.mark.skip
+def test_save(model, preds, output_path):
```

Comment:
Pytest will [treat any function starting with `test_` or ending with `_test` as a test](https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#test-discovery). If you would like to define a helper function without it being considered a test, then renaming it works as well.